input/text/src/commonMain/kotlin/majestic/Search.kt

```diff
@@ -54,6 +56,7 @@ fun Search(
 ) {
     var isFocused by remember { mutableStateOf(false) }
     var containerWidth by remember { mutableStateOf(0) }
+    val orientation = rememberScreenOrientation()
```

Comment:
You are still calling `rememberScreenOrientation()`, what about what we spoke about?